lib/neo4j/active_node/query/query_proxy.rb

```diff
@@ -186,11 +186,8 @@ def <<(other_node)
         #
         # @return [QueryProxy] A new QueryProxy
         def branch(&block)
-          if block
-            instance_eval(&block).query.proxy_as(self.model, identity)
-          else
-            fail LocalJumpError, 'no block given'
-          end
+          fail LocalJumpError, 'no block given' unless block
```

Comment:
@cheerfulstoic 
Rubocop 0.4 doesn't agree with you 
Just joking, I understand what you're saying 😜
I changed that thing because of a suggestion of it from my Editor, but no problem about restoring it ;)